tfjs-backend-webgl/src/kernels/Conv2D_impl.ts

```diff
@@ -39,6 +39,41 @@ type Conv2DConfig = {
   activation?: backend_util.Activation
 };
 
+function fitPreluActivationWeightsIntoNhwcFormat(
+    alpha: TensorInfo, outputShape: [number, number, number, number],
+    isChannelsLast: boolean, backend: MathBackendWebGL) {
+  // PReLU's activation weights could be a scalar, a 1-D tensor or a 3-D
+  // tensor.
+  const alphaShape = alpha.shape;
+  util.assert(
+      alphaShape.length <= 1 || alphaShape.length === 3,
+      () => `WebGL conv2d only supports scalar, 1-D Tensor or 3-D ` +
+          `Tensor PReLU activation weights but got a tensor of ` +
+          `rank-${alphaShape.length}.`);
+
+  if (alphaShape.length === 1) {
```

Comment:
you can use assertAndGetBroadcastShape method to validate the shape compatibility for broadcasting.

Reply:
Done.